app/migrations/Version20240205082236.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+use Ivoz\Core\Infrastructure\Persistence\Doctrine\LoggableMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20240205082236 extends LoggableMigration
+{
+    public function getDescription(): string
+    {
+        return 'Add active field to Administrator';
+    }
+
+    public function preUp(Schema $schema): void
```

Comment:
FYI: We haven't use preUp before so I'm not sure whether this is going to be recorded into changelog or not. Apart from that, it looks like a good method to make this update

Reply:
There is a record for it in the Changelog table. However, it is not associated with the entity:

>    id: be065ad1-6060-4fd5-952c-8d9edfe039f4
>    entity: unkown
>  entityId: 0
>      data: {"query":"UPDATE Administrators SET email = \"noreplay@irontec.com\" WHERE id = 1 AND email IS NULL","arguments":[]}
> createdOn: 2024-02-05 11:36:25
> microtime: 7541
> commandId: f6fd25e9-44e0-4f56-938a-f42523569366

Is this enough?

Reply:
Yes. We don't parse "raw" sql statements